src/main/java/com/kelinci/mainapp/api/UserController.java

```diff
@@ -71,9 +76,14 @@ public void confirm(@RequestBody ConfirmationRequest confirmation) {
                 //tutaj jedyne co wykombinowalem to usunac usera z listy, ustawic mu setConfirmed na true, i potem dodac ponownie do listy
                 //bo na to wygląda, że zmiana pól obiektu nie updatuje tych pól jeśli obiekt jest już w liście - dobrze wiedziec
             }
-            confirmedUser = userToBeConfirmed;
+            ourUser = userToBeConfirmed;
         }
+    }
 
+    private static String generateMailCode() {
+        Integer random = (int) (1000000 * Math.random());
+        String mailCode = Integer.toString(random);
```

Comment:
Mnie tu IntelliJ świeci, w paru miejscach na sraczkowo żółto - a Tobie? :)

Reply:
ok, zrobiłem sugestie - wrzuciło return Integer.toString(random)